contrib/fedora-minimal/Dockerfile

```diff
@@ -1 +1 @@
-FROM registry.fedoraproject.org/fedora-minimal:latest
+FROM registry.fedoraproject.org/fedora-minimal:34
```

Comment:
Even `:34` seems risky to me -- that, too, could be overridden. Actually, why do we even do this rebuild thing anyway? It seems the sort of thing that should require human validation.

Reply:
I agree.  The best solution imho would be to not auto-build on Quay, and just push one image to quay and leave it there without touching it anymore.

A quick check suggests that F34 solves the issue.